docker/dockerfiles/Dockerfile.onnx.gpu

```diff
@@ -28,7 +28,9 @@ COPY requirements/requirements.sam.txt \
     ./
 
 RUN python3 -m pip install -U pip
-RUN python3 -m pip install --extra-index-url https://download.pytorch.org/whl/cu118 \
+RUN python3 -m pip install \
+    --extra-index-url https://download.pytorch.org/whl/cu118 \
+    --extra-index-url https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/onnxruntime-cuda-11/pypi/simple/ \
```

Comment:
what is this guy doing? changing the default cuda version for onnxruntime? so onnxruntime installs cuda?

Reply:
The default is now to install the onnxruntime that's compatible with CUDA 12; this forces it to install the one built for CUDA 11 instead. (It used to be the reverse; you had to choose the channel for CUDA 12 and the default was 11)